src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReader.WinMD.cs

```diff
@@ -11,15 +11,7 @@ public partial class MetadataReader
     {
         internal const string ClrPrefix = "<CLR>";
 
-        internal static readonly byte[] WinRTPrefix = new[] {
-            (byte)'<',
-            (byte)'W',
-            (byte)'i',
-            (byte)'n',
-            (byte)'R',
-            (byte)'T',
-            (byte)'>'
-        };
+        internal static readonly byte[] WinRTPrefix = "<WinRT>"u8.ToArray();
```

Comment:
does this still benefit from the compiler's optimization to burn the content into the IL, given you're now doing ToArray?

Reply:
The primary benefit here is brevity / readability. This will still allocate an array.